src/quo/components/wallet/progress_bar/component_spec.cljs

```diff
@@ -75,7 +75,7 @@
                     :width           8
                     :borderRadius    3
                     :borderColor     colors/neutral-80-opa-5
-                    :backgroundColor colors/success-50})))
+                    :backgroundColor (colors/resolve-color :success theme 50)})))
```

Comment:
@mmilad75 please take a look at the api for resolve color. the 50 & 60 values you are using here are incorrect. For example. you are now setting 50% opacity here.
before `custom-color` function was using the 50/60 suffixes to resolve the color. Instead `resolve-color` function uses theme.
So for example 
`(colors/custom-color :blue 50)` is equal to `(colors/custom-color :blue :light)`

There are some tests in that colors file which might make this a bit clearer too.

Reply:
Done